src/util/BaseLoginRouter.js

```diff
@@ -250,6 +250,11 @@ function (Okta, BrowserFeatures, Settings,
                   }
                 });
                 this.appState.unset('flashError');
+                this.controller.trigger(
```

Comment:
This `flashError` happens only when the `stateToken` expires, and is used only in this place. However, this requires us to trigger another `afterError` event with the cached transaction error.

Therefore, if a user's state times out, we're going to fire this event two times (see [`LoginRouter_spec`](https://github.com/okta/okta-signin-widget/pull/594/files#diff-1fa99203134b9819901ba981b75f4af7)).

Reply:
why would trigger `afterError` twice at this case sounds fine?
I wonder if we shall only trigger `afterError` once.?